Frontend/src/pages/registration_edit/components/RegistrationEditor.jsx

```diff
@@ -1,35 +1,43 @@
 import { EventSelector } from '@thewca/wca-components'
-import React, { useEffect, useState } from 'react'
+import React, { useContext, useEffect, useState } from 'react'
 import { Button, Checkbox, TextArea } from 'semantic-ui-react'
-import { useHeldEvents } from '../../../api/helper/hooks/use_competition_info'
-import useCompetitorInfo from '../../../api/helper/hooks/use_competitor_info'
+import { CompetitionContext } from '../../../api/helper/context/competition_context'
 import { getSingleRegistration } from '../../../api/registration/get/get_registrations'
 import { updateRegistration } from '../../../api/registration/patch/update_registration'
 import { setMessage } from '../../../ui/events/messages'
 import LoadingMessage from '../../../ui/messages/loadingMessage'
 import styles from './editor.module.scss'
+import { useQuery } from '@tanstack/react-query'
+import getCompetitorInfo from '../../../api/user/get/get_user_info'
 
 export default function RegistrationEditor({ user_id, competition_id }) {
```

Comment:
Can/should we get the user and competition info from the contexts/params like we do everywhere else?

Reply:
yeah, changed everywhere